plugins/lsp/language-servers/default.nix

```diff
@@ -168,6 +168,11 @@ with lib; let
       description = "Enable elmls, for Elm.";
       package = pkgs.elmPackages.elm-language-server;
     }
+    {
+      name = "emmet-ls";
```

Comment:
Isn't it prefereable to have `emmet_ls` like upstream nvim-lspconfig? It's easier to adapt a normal neovim config

Reply:
I am ok with that, but it is not the convention we have sued until now.